### DISCUSSION TITLE
use `ps ah -e` in `get_processes()`?

### DISCUSSION THREAD

**Discussion by yetyetanotherusername** [mfussenegger/nvim-dap#988] [Ideas]
Hey,

I'm not quite sure how `ps ah` works on other systems but on Fedora, it returns only a handful of processes related to the terminal emulator it is called from. This makes it impossible for me to attach to the application I need to debug using `pick_process`, since it never shows up in the list of processes.

I'm wondering if anybody can see any downsides to adding the `-e` parameter? Perhaps the less verbose `ps a -u "$USER"`?

**Comment by mfussenegger** [comment]
Changed it with https://github.com/mfussenegger/nvim-dap/pull/991
